docs/api.md

```diff
@@ -386,11 +386,11 @@ those common validations are defined here.
 * `schema`: {
     * `id`: isA.string.length(32).regex(HEX_STRING)
     * `location`: isA.object({
-      * `city`: isA.string.optional.allow(null)
-      * `country`: isA.string.optional.allow(null)
-      * `state`: isA.string.optional.allow(null)
-      * `stateCode`: isA.string.optional.allow(null)
-      * })
+    * `city`: isA.string.optional.allow(null)
+    * `country`: isA.string.optional.allow(null)
+    * `state`: isA.string.optional.allow(null)
+    * `stateCode`: isA.string.optional.allow(null)
+    * })
```

Comment:
Did the generated output actually change here somehow, or is this manually edited? If it's the latter, the manual edits are going to be lost fwiw (and I also don't really understand why they're here in that case).